portfolio/src/main/java/com/google/sps/servlets/ListMessagesServlet.java

```diff
@@ -42,14 +42,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    long value = Integer.parseInt(request.getParameter("value"));
```

Comment:
1. What happens if `value` can not be parsed as int?
2. It is unclear what `value` represents here, is there a more descriptive name for the variable? I see that the parameter is called value as well, so a comment might be an alternative.